src/util/floatingpoint.cpp

```diff
@@ -36,4 +36,307 @@ void FloatingPointLiteral::unfinished (void) const {
   Unimplemented("Floating-point literals not yet implemented.");
 }
 
+  FloatingPoint::FloatingPoint (unsigned e, unsigned s, const BitVector &bv) :
+    fpl(e,s,0.0),
+    t(e,s) {}
+
+
+  static FloatingPointLiteral constructorHelperBitVector (const FloatingPointSize &ct, const RoundingMode &rm, const BitVector &bv, bool signedBV) {
```

Comment:
Hint for other reviewers of this PR: this code is changing in future commits. I was confused why this function always returns the same thing :)

Reply:
Yes, sorry, I should have said.  These are mostly dummy implementations.  The patch is about fixing the interface (which matters for several later patch sets) an actual implementation comes much later as it needs symfpu.